tcf_website/static/icons/icons.html

```diff
@@ -0,0 +1,8 @@
+<i class="fas fa-user"></i> Instructor
```

Comment:
To be clear, is this file just for keeping track of which icon we're using for each field? Can't find any references to it anywhere else in the code.

Reply:
Yep!